pages/linux/pacman-sync.md

```diff
@@ -0,0 +1,36 @@
+# pacman sync
```

Comment:
Didn't we decide to use `pacman --sync` as the title?
```suggestion
# pacman --sync
```

Reply:
I think you're thinking of the filename?

Reply:
No, I'm not, actually.  Don't you think it would be better to have the title match what is actually entered in the terminal?  Currently it looks like `sync` is a normal subcommand of `pacman`.

Reply:
Oh I see what you mean. Yes, I agree - this change would be a good idea.